.gitignore

```diff
@@ -0,0 +1,2 @@
+__pycache__/
+venv/
```

Comment:
Is this neccessary?

Reply:
This was only required on my system, but it should be useful for anyone that wants to work on this project.